packages/core/core/src/worker.js

```diff
@@ -1,45 +1,43 @@
 // @flow strict-local
 
-import type {ParcelOptions, AssetRequest, JSONObject} from '@parcel/types';
+import type {ParcelOptions, AssetRequest} from '@parcel/types';
 import type {Bundle} from './types';
 import type BundleGraph from './BundleGraph';
 
 import TransformerRunner from './TransformerRunner';
 import PackagerRunner from './PackagerRunner';
 import Config from './Config';
 
-type Options = {|
-  config: Config,
-  options: ParcelOptions,
-  env: JSONObject
-|};
-
-let transformerRunner: TransformerRunner | null = null;
-let packagerRunner: PackagerRunner | null = null;
+export function init() {}
 
-export function init({config, options}: Options) {
-  transformerRunner = new TransformerRunner({
-    config,
-    options
-  });
-  packagerRunner = new PackagerRunner({
+export function runTransform({
+  request,
+  config,
+  options
+}: {
+  request: AssetRequest,
+  config: Config,
+  options: ParcelOptions
+}) {
+  return new TransformerRunner({
```

Comment:
wonder how costly constructing these objects is. maybe we could cache them somehow if it becomes a problem...

Reply:
Pretty cheap as far as I can tell. Just creating new objects and simple property assignment in the constructors. Nothing else really happens between that and performing the work.